cfpb/consumerfinance.gov
# Fixing 404s in Your Financial Path to Graduation

Two hyperlinks on the Your Financial Path to Graduation tool were pointing to "link" instead of a proper URL, causing both to 404. This PR replaces those items, which are part of the hard-coded sections of that tool instead of within Wagtail, with appropriate links.

---




## Changes

- Two link refs on the `10-max-debt-guideline.html` file


## How to test this PR

1. localhost
2. /paying-for-college/your-financial-path-to-graduation/
3. Select "Get started", then fill out the form until you are able to navigate to "Affording your loans" and "Is this too much debt?"
4. Under "Current debt from prior schooling" confirm that the two links in the helper text resolve correctly


## Screenshots




## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
